cruzdb/__init__.py

```diff
@@ -2,6 +2,7 @@
 cruzdb: library for pythonic access to UCSC genome-browser's MySQL database
 """
 from __future__ import print_function
+from past.builtins import long
```

Comment:
This needs a guard for version
```
$ python2.7 -c "from past.builtins import long"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named past.builtins
```

Reply:
I didn't get that error with python 2.7.15, but perhaps mine came with the `past.builtins `module pre-installed. Nevertheless, I added a version check before loading the module, as it indeed is not required for python 2.